### PR TITLE
Update datasource documents LB2

### PR DESCRIPTION
**Pull request by ssh24**
### Description

Update api documents/ js docs and address all REVIEW and FIXME tags
#### Related issues

connect to https://github.com/strongloop-internal/scrum-apex/issues/175

### Checklist

- [ ] New tests added or existing tests modified to cover all changes N/A
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Review by 0candy — APPROVED** [review]

File: pages/en/lb2/Attaching-models-to-data-sources.md [hunk 1]
```diff
@@ -81,6 +98,14 @@ you can specify the data source you want it to use from among those you've added
 [Data source generator](Data-source-generator.html) and the default `db`
 data source (that uses the [memory connector](Memory-connector.html)).
 
+```shell
```
Comment (dhmlau) [inline pages/en/lb2/Attaching-models-to-data-sources.md]: might wanna add something like:
to create a model, run `slc loopback:model` or `apic create --type model` (in similar format you have above)

**Review by dhmlau — APPROVED** [review]
LGTM 
you might wanna address my comment.

File: pages/en/lb2/datasources.json.md [hunk 2]
```diff
@@ -80,17 +80,17 @@ To connect a model to a data source, follow these steps:
...
     $ slc loopback:datasource
     ? Enter the data-source name: mysql-corp
-    ? Select the connector for mysql: MySQL (supported by StrongLoop)
+    ? Select the connector for mysql-corp: MySQL (supported by StrongLoop)
     ```
 
-    Follow the prompts to name the datasource and select the connector to use.
-    This adds the new data source to `datasources.json`
+    Follow the prompts to fill in the connector configurations.
+    This adds the new data source to `server/datasources.json` which can be later used to update the config if needed.
```
Comment (crandmck) [inline pages/en/lb2/datasources.json.md]: We use terminology "data source properties" not "config/configuration" or "settings".  Technically, they are for the data source (i.e. this instance), not the connector (the `loopback-connector-*` package).

> Follow the prompts to fill in the connector configurations.

Change to:
"Respond to the prompts to set data source properties such as database credentials."

> ... which can be later used to update the config if needed

Change to:

"To change the data source properties, edit this file."

**Comment by crandmck** [comment]
Please make changes here corresponding to the requested changes for LB3 in #369.

**Review by crandmck — CHANGES_REQUESTED** [review]

**Review by crandmck — APPROVED** [review]

**Pull request merged**